modin/core/io/column_stores/column_store_dispatcher.py

```diff
@@ -158,11 +160,14 @@ def build_columns(cls, columns):
             List with lengths of `col_partitions` subarrays
             (number of columns that should be read by workers).
         """
+        columns_length = len(columns)
+        if columns_length == 0:
+            return [], []
         num_partitions = NPartitions.get()
         column_splits = (
-            len(columns) // num_partitions
-            if len(columns) % num_partitions == 0
-            else len(columns) // num_partitions + 1
+            columns_length // num_partitions
+            if columns_length % num_partitions == 0
+            else columns_length // num_partitions + 1
         )
         col_partitions = [
             columns[i : i + column_splits]
```

Comment:
Can the line below be changed to `columns_length` as well?

Reply:
Done.